### ISSUE TITLE
ETA on Splunk Cloud Support?

### ISSUE THREAD

**Issue by BeanBagKing** [microsoftgraph/security-api-solutions#36]
# Is support for Splunk Cloud coming? Is there an ETA on availability?

# Additional Info
This may also require updates to make the app compatible with Python 3 (if it isn't already).

# General
The post below is currently about four months old and references working on support for Splunk Cloud. The add on in Splunk Base appears to have been added about 6 months ago, but no updates have been made since initial release. If there is an update forthcoming soon, I would like to know so that we can hold on further development until this is released. However, if there is no update coming soon, then it would help me to know this so that I can make alternative plans.

https://techcommunity.microsoft.com/t5/security-privacy-and-compliance/introducing-the-new-microsoft-graph-security-api-add-on-for/ba-p/815972

**Comment by BeanBagKing** [comment]
It appears that this was fixed yesterday (30 Jan 2020). The release notes note that Splunk Cloud is now supported, alongside Python 3. However, the compatibility notes to the right have not been updated and I have not tested this with Splunk Cloud yet. If anyone can confirm that they now have this working with Splunk Cloud, I'll close this issue. Otherwise, I'll self-close once we have indicated that it is working.
